Documentation/API Backend/openrpc/components/schemas/state.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "object",
+    "properties": {
+        "robotOnline": {
+            "type": "boolean"
+        },
+        "jointPositions": {
+            "$ref": "#/components/schemas/jointPositions"
```

Comment:
the getState method provides the ability to include or exclude keys. You can also call the getPosition method to get just that. The reason that all state items are also included in a state object is so that there is a place to list all of the state parameters and it is a mechanism for transmitting any changes to state.

Reply:
It is a way to generalize the API. So we dont' necessarily have to create methods for every parameter that gets added to state. If you add a new key to state, it can automatically be subscribed to.